### PR TITLE
Add example LTI 1.1 configuration and readme

### PR DESCRIPTION
**Pull request by jgwerner**
- Adds an example LTI 1.1 configuration file in a new examples directory
- Adds a README to use the example with `virtualenv`

#53

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# LTI Authenticator Examples
+
```
Comment (consideRatio) [inline examples/README.md:2]: Perhaps you could add a sentence here that describes the outcome of following all steps here or similar? I find myself often working through examples to figure out what the example actually is about.

I understand this is an example procedure to deploy JupyterHub locally, configured to authenticate against a LTI v1.1 identity provider, compared to the latest version v1.3?

**Review by consideRatio — APPROVED** [review]
This LGTM overall, left a comment to summarize what the example was about, but other than that +1 for merge!

**Comment by consideRatio** [comment]
Is it correct that this PR doesn't close #53, but that it instead is a way to debug #53 at this point?

**Comment by jgwerner** [comment]
@consideRatio good point - I'll update the PRs description to not close the issue automatically.

**Comment by jgwerner** [comment]
@consideRatio I opened #56 to help fix some testing issues with the file provided in the examples folder. If possible, we should merge `56` then I could rebase this one (`54`) so that GH Actions are green.

File: examples/README.md (hunk 1 above)
Comment (jgwerner) [reply examples/README.md:2]: I updated the readme for clarity. Your assumption is almost correct, it doesn't necessarily compare LTI 1.1 with LTI 1.3, rather it provides working configuration examples so users can test different settings with the LTI authenticators. Once we merge #39 then we can include examples for LTI 1.3 as well.

**Review by consideRatio — APPROVED** [review]

**Comment by jgwerner** [comment]
@consideRatio rebase was completed and tests are passing now. Let me know if you would like me to merge.

**Pull request merged**